Makefile

```diff
@@ -0,0 +1,72 @@
+clean:
+	rm -rf bazel-*
+	rm -rf man
+	rm -rf docs
+
+build:
+	bazel build //...
+
+test:
+	bazel test //...
+
+.PHONY: clean build test
+
+bazel-bin/verilog/tools/lint/verilog_lint:
+	bazel build //verilog/tools/lint:verilog_lint
+
+bazel-bin/verilog/tools/syntax/verilog_syntax:
+	bazel build //verilog/tools/syntax:verilog_syntax
+
+bazel-bin/verilog/tools/formatter/verilog_format:
+	bazel build //verilog/tools/formatter:verilog_format
+
+tools: bazel-bin/verilog/tools/lint/verilog_lint bazel-bin/verilog/tools/syntax/verilog_syntax bazel-bin/verilog/tools/formatter/verilog_format
+	@true
+
+# Generate man pages from the tools
+man/verilog_lint.1: bazel-bin/verilog/tools/lint/verilog_lint
```

Comment:
It would be cool if we could describe these just in a single generic rule (`man/%.1 : ...`), but unfortunately, these tools have `/lint/`, `/syntax/` and `/formatter/` in their path, which makes them escape the common pattern matching rules in make. So I guess no dice.